### ISSUE TITLE
addPathsIfNotIncluded() breaks standalone version

### ISSUE THREAD

**Issue by hagenw** [TWOEARS/auditory-front-end#12]
The AFE was designed to be used without all other parts of Two!Ears (including [main](https://github.com/TWOEARS/main)).
The function `addPathsIfNotIncluded()` introduced in #7 breaks this as this function is only available in [main](https://github.com/TWOEARS/main).
I see two solutions to this problem:
1) Simply copy the code from `addPathsIfNotIncluded()` into `startAuditoryFrontEnd()`
2) Add the function to the AFE as well

I would be in favor of the first approach. Is this fine, or would there be further problems?

**Comment by ivo--t** [comment]
I support your approach (1). 

This could also be the solution to integrating this fix into the binsim, @fietew ? I guess you haven't merged my respective pull request exactly because of this reason?

**Comment by hagenw** [comment]
I created the branch `addpathes` where I changed `startAuditoryFrontEnd` <a href="https://github.com/TWOEARS/auditory-front-end/blob/addpathes/startAuditoryFrontEnd.m">accordingly</a>.
During those changes I stumbled over two other things:
1) The `test/` folders are normally not included in the start scripts. @Hardcorehobel: was there a special reason why it was included in `startAuditoryFrontEnd`? In my changes I removed the folder.
2) The path creation with `strsplit( genpath(fullfile(basePath, 'src')), pathsep )` returns `''` which cannot be found in the current pathes, so `addpath` was still called every time you call `startAuditoryFrontEnd`. I changed this by using explicitly path names in `startAuditoryFrontEnd`. This might have the downside that you add a new folder to `src/` and forget to add it to `startAuditoryFrontEnd` as well. So, maybe there is a better solution?
